### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/advanced/additional-responses.md`

### PR DESCRIPTION
**Pull request by jaystone776**
as title

### CODE REVIEW

File: docs/zh/docs/advanced/additional-responses.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# OpenAPI 中的附加响应
+
+!!! warning "警告"
+
+    本章较难。
+    
+    **FastAPI** 新手可跳过本章。
+
+本章学习使用附加状态码、媒体类型、描述等声明响应。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:9]:
```suggestion
本章学习为响应声明额外的状态码、媒体类型和描述。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+这些响应会包含在 OpenAPI 概图里，并在 API 文档中显示。
+
+但必须要确保这些响应与状态码和 `content` 一起直接以 `JSONResponse` 等方式返回。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:13]:
```suggestion
但必须要确保这些响应与状态码和 `content` 一起直接以一个 `Response` 对象如 `JSONResponse` 返回。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+## 使用 `model` 的响应
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:15]:
```suggestion
## 以 `model` 声明额外响应
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
+# OpenAPI 中的附加响应
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:1]:
```suggestion
# OpenAPI 中的额外响应
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+这些响应会包含在 OpenAPI 概图里，并在 API 文档中显示。
+
+但必须要确保这些响应与状态码和 `content` 一起直接以 `JSONResponse` 等方式返回。
+
+## 使用 `model` 的响应
+
+*路径操作装饰器*支持 `response` 参数。
+
+`response` 参数可以接收键是响应的状态码，如 `200`，值是响应信息的字典。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:19]:
```suggestion
`responses` 参数可以接收以响应状态码为键，如 `200`，响应信息为值的字典。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+
+响应字典可以包含 Pydantic 模型的键 `model`，如 `response_model`。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:21]:
```suggestion
响应信息本身是个字典，可以包含名为 `model` 的键，其值为和 `response_model` 类似的 Pydantic 模型。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+
+!!! note "笔记"
+
+    注意，必须直接返回 `JSONResponse`。
+
+!!! info "说明"
+
+    `model` 键不是 OpenAPI 的组件。
+    
+    **FastAPI** 在此接收 Pydantic 模型，生成 `JSON Schema`，并把它推送到正确的位置。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:39]:
```suggestion
    **FastAPI** 在此接收 Pydantic 模型，生成 `JSON Schema`，并把它置于正确的位置。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## 主响应的附加媒体类型
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:168]:
```suggestion
## 主响应的额外媒体类型
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+
+本章学习使用附加状态码、媒体类型、描述等声明响应。
+
+这些响应会包含在 OpenAPI 概图里，并在 API 文档中显示。
+
+但必须要确保这些响应与状态码和 `content` 一起直接以 `JSONResponse` 等方式返回。
+
+## 使用 `model` 的响应
+
+*路径操作装饰器*支持 `response` 参数。
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:17]:
```suggestion
*路径操作装饰器*支持 `responses` 参数。
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+
+**FastAPI** 支持合并 `response_model`、`status_code`、`responses` 参数等多个位置的响应信息。
+
+使用默认状态码 `200`（或自定义状态码），声明 `response_model`。然后，直接在 OpenAPI 概图中，为 `responses` 中的同一响应声明更多信息。
+
+**FastAPI** 保留 `responses` 中的信息，并把它与模型的 JSON 概图合并在一起。
+
+例如，使用 Pydantic 模型声明包含状态码 `404` 的响应，并包含自定义 `description`。
+
+带状态码 `200` 的响应不仅使用 `response_model`，还包含了自定义 `example`：
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:198]:
```suggestion
以及使用 `response_model` 状态码为 `200` 的响应，但是还包含了自定义的 `example`：
```

File: docs/zh/docs/advanced/additional-responses.md [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+
+OpenAPI 中会合并、包含这些内容，然后显示在 API 文档中：
+
+<img src="/img/tutorial/additional-responses/image01.png">
+
+## 合并预定义响应与自定义响应
+
+**FastAPI **还可以合并*路径操作*中的预定义响应和自定义响应。
+
+此处使用 Python **字典解包**，即 `**dict_to_unpack`：
```
Comment (waynerv) [inline docs/zh/docs/advanced/additional-responses.md:212]:
```suggestion
此处使用 Python **字典解包**，即通过 `**dict_to_unpack` 语法解包一个 `dict`：
```

**Review by waynerv — CHANGES_REQUESTED** [review]

**Comment by UP929312** [comment]
Why did you make 400 pull requests instead of having them all in one?

**Comment by thunderamur** [comment]
> Why did you make 400 pull requests instead of having them all in one?

The Chinese bureaucracy requires the number of pull requests. I think the maintainers should reject all this spam.

**Review by aqcool — APPROVED** [review]
ok

**Comment by tiangolo** [comment]
📝 Docs preview for commit 70b2f42b94486e09164f6b4b0f24da864f41ebf4 at: https://6499e381a2e6b304e1ddcb48--fastapi.netlify.app

**Comment by tiangolo** [comment]
Thank you @jaystone776! It seems this was covered by https://github.com/tiangolo/fastapi/pull/10325, so I'll now close this one.

But if you feel the current one can be improved, feel free to add more PRs. ☕

**Pull request closed without merging**